rfc/graphql-schema/additionalTypeDefs/byPage/invoices.ts

```diff
@@ -0,0 +1,143 @@
+export default /* GraphQL */ `
+  enum InvoiceStatus {
+    OPEN
+    PAID
+    OVERDUE
+    CANCELLED
+  }
+
+  type InvoiceStats {
+    openBalance: Float!
+    overdueBalance: Float!
+  }
+
+  input InvoiceDateRangeFilterInput {
+    from: DateTime!
+    to: DateTime
+  }
+
+  input InvoiceFiltersInput {
+    search: String
+    dateRange: InvoiceDateRangeFilterInput
+    status: InvoiceStatus
+  }
+
+  enum InvoicesSortInput {
+    INVOICE_NUMBER_A_TO_Z
+    INVOICE_NUMBER_Z_TO_A
+    ORDER_ID_A_TO_Z
+    ORDER_ID_Z_TO_A
+    CREATED_AT_NEWEST
+    CREATED_AT_OLDEST
+    DUE_DATE_NEWEST
+    DUE_DATE_OLDEST
+    ORIGINAL_BALANCE_HIGHEST
+    ORIGINAL_BALANCE_LOWEST
+    OPEN_BALANCE_HIGHEST
+    OPEN_BALANCE_LOWEST
+  }
+
+  type CreateCartFromInvoiceError implements Error {
+    message: String!
+  }
+
+  type CartCreateResult {
+    cart: Cart
+    errors: [CreateCartFromInvoiceError!]!
+  }
+
+  type GenerateInvoicePdfError implements Error {
+    message: String!
+  }
+
+  type GenerateInvoicePdfResult {
+    url: String
+    errors: [GenerateInvoicePdfError!]!
+  }
+
+  type Invoice {
+    id: ID!
+    invoiceNumber: String!
+    createdAt: DateTime!
+    dueDate: DateTime!
+    openBalance: Money!
+    originalBalance: Money!
```

Comment:
worth its own namespace (`balance.original`) or we confident that these two will be it for a while?

Reply:
I do like the `balance` property, I moved both elements inside. I don't think we'll have anything else in the future, but 🤷